ansible_base/rbac/models.py

```diff
@@ -30,9 +30,13 @@
 class DABPermission(models.Model):
     "This is a minimal copy of auth.Permission for internal use"
 
-    name = models.CharField("name", max_length=255)
-    content_type = models.ForeignKey(ContentType, models.CASCADE, verbose_name="content type")
-    codename = models.CharField("codename", max_length=100)
+    name = models.CharField("name", max_length=255, help_text=_("The name of this permission."))
+    content_type = models.ForeignKey(ContentType, models.CASCADE, verbose_name="content type", help_text=_("The content type this permission will apply to."))
```

Comment:
Let's say we have a permission statement "John can pet the dog named Fido". That would be an instantiation of `Permission` (note that a permission entry it not an instantiation), the content type of the permission being dog.

Reply:
Writing more comments further down, I realize "subject" is a hazardous word. "object" might be better. English SVO grammar structure would imply "subject" is what is referred to as "actor" elsewhere.